app/helpers/finance_helper.rb

```diff
@@ -20,4 +34,17 @@ def total_payment(breakdown)
   def total_vat(breakdown, lead_provider)
     total_payment(breakdown) * (lead_provider.vat_chargeable ? 0.2 : 0.0)
   end
+
+  def payment_period
+    index = MILESTONE_DATES.each_index.detect { |date| Time.zone.today.before?(Date.parse(MILESTONE_DATES[date])) }
+    MILESTONE_DATES.slice(index - 1, 2)
+  end
+
+  def pretty_payment_period
```

Comment:
Looks quite stateless, I'd probably try to be more explicit and define this as a module method (see https://thoughtbot.com/blog/disambiguate-rails-helpers).
100% personal taste though, feel free to ignore.